### PR TITLE
enable Windows build.

### PR DESCRIPTION
**Pull request by seanyen**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by seanyen** [comment]
@conda-forge-admin, please rerender

**Review by gartung — APPROVED** [review]

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,14 +11,16 @@ source:
...
-  number: 0
-  skip: true  # [win]
+  number: 1
+  skip: true  # [win and vc<14]
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - cmake
```
Comment (chrisburr) [inline recipe/meta.yaml:21]: It's not related to this PR but this should have already been there instead relying on the host's `make`
```suggestion
    - cmake
    - make  # [unix]
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -11,14 +11,16 @@ source:
...
+  run:
+    - vs2015_runtime  # [win]
```
Comment (chrisburr) [inline recipe/meta.yaml]: Is this needed? Naively I expect it to be included automatically with `run_exports` but I'm not familiar enough with Windows to know for sure.

**Review by chrisburr — COMMENTED** [review]
Thanks @seanyen! 

I have two minor points then I'll merge.

File: recipe/meta.yaml (hunk 2 above)
Comment (seanyen) [reply recipe/meta.yaml]: Thanks for the pointer! I am new to conda build system and I don't know about the `run_exports` thing enough, but it seems the build\install still to work fine without defining the explicit dependency, so I removed them.

**Comment by seanyen** [comment]
@conda-forge-admin, please rerender

**Comment by chrisburr** [comment]
Thank you @seanyen! 🍰

**Pull request merged**